packages/sdks/snippets/svelte/src/components/LivePreview.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import { onMount, onDestroy } from 'svelte';
+  import { fetchOneEntry, subscribeToEditor } from '@builder.io/sdk-svelte';
+  import type { BuilderContent } from '@builder.io/sdk-svelte';
+  let content: BuilderContent | null = null;
+  let isLoading = true;
+  let unsubscribe: (() => void) | null = null;
+
+  let urlPath = window.location.pathname;
+
+  onMount(async () => {
+    try {
+      const data = await fetchOneEntry({
+        model: 'blog-data',
+        apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+        userAttributes: {
+          urlPath,
+        },
+      });
+      content = data;
+    } catch (err) {
+      console.error('Error fetching Builder content:', err);
+    }
+
+    isLoading = false;
+
+    unsubscribe = subscribeToEditor({
+      model: 'blog-data',
+      apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+      callback(updatedContent) {
+        content = updatedContent;
+      },
+    });
+  });
+
+  onDestroy(() => {
+    if (unsubscribe) {
+      unsubscribe();
+    }
+  });
```

Comment:
in svelte, you can destroy in the return value of `onMount`, so you don't need to store `unsubscribe and use `onDestroy`.

```suggestion
  let urlPath = window.location.pathname;

  onMount(async () => {
    try {
      const data = await fetchOneEntry({
        model: 'blog-data',
        apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
        userAttributes: {
          urlPath,
        },
      });
      content = data;
    } catch (err) {
      console.error('Error fetching Builder content:', err);
    }

    isLoading = false;

    unsubscribe = subscribeToEditor({
      model: 'blog-data',
      apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
      callback(updatedContent) {
        content = updatedContent;
      },
    });
    
    return () => unsubscribe()
  });
```